vars/emailUtils.groovy

```diff
@@ -0,0 +1,27 @@
+/**
+ * Requires "Email Notifications Plugin"
+ */
+
+private def sendEmail(sentTo, replyTo, subject, content) {
+
+        if(sentTo != null && !sentTo.isEmpty()) {
+          emailext(body: content, mimeType: 'text/html',
+             replyTo: '$DEFAULT_REPLYTO', subject: subject,
```

Comment:
Shouldn't `'$DEFAULT_REPLYTO'` just be `replyTo` since you'll be passing in the value of `replyTo` from the `call` function?

Reply:
Fixed.